playbooks/roles/airship-setup-deployer/tasks/main.yml

```diff
@@ -37,12 +37,17 @@
   tags:
     - install
 
+- name: Set the user name
+  set_fact:
+    user_name: "{{ lookup('env','USER') }}"
+
 - name: Add the user to the group 'docker'
   become: yes
   user:
-    name: "{{ lookup('env','USER') }}"
+    name: user_name
     groups: docker
     append: yes
+  when: user_name
```

Comment:
Instead of skipping this step if USER is unset, we should make sure that USER is set to the appropriate value during the jenkins job

Reply:
I agree